diff --git a/node_modules/puppeteer-extra/dist/index.js b/node_modules/puppeteer-extra/dist/index.js
index 1da5096..81b782c 100644
--- a/node_modules/puppeteer-extra/dist/index.js
+++ b/node_modules/puppeteer-extra/dist/index.js
@@ -19,7 +19,7 @@ const deepmerge_1 = __importDefault(require("deepmerge"));
  * @implements {VanillaPuppeteer}
  *
  * @example
- * const puppeteer = require('puppeteer-extra')
+ * const puppeteer = require("chrome-aws-lambda");
  * puppeteer.use(require('puppeteer-extra-plugin-anonymize-ua')())
  * puppeteer.use(require('puppeteer-extra-plugin-font-size')({defaultFontSize: 18}))
  *
